ci_tests.sh

```diff
@@ -3,5 +3,14 @@ set -ex
 
 export MONTAGU_PROXY_TESTS=montagu-proxy-tests
 
+# If we do not have a node_modules dir (which will be the case on teamcity agent), create a volume so we can access
```

Comment:
Injecting js from npm worked like a dream locally but of course it failed on TeamCity because the agent doesn't have any node modules. Not sure if this is the standard way to go about this, but here I'm saying that if we don't already have a node_modules dir on the host (ie if we're on TC rather than running locally) then create a volume which will pull out the node modules from the test container to a local dir, so that they're available for the proxy image to pull out the bits of js it wants later.

This also means that I had to change around the test container a bit, delaying npm install until the entry point because otherwise the node_modules were overwritten by the host's nonexistent modules...

Reply:
Ah, so this is why we take the docker in docker approach elsewhere. We could actually repurpose your test container here to be a build environment with relatively few code changes: https://github.com/vimc/montagu-proxy/pull/23